internal/formats/common/spdxhelpers/to_syft_model.go

```diff
@@ -24,7 +25,13 @@ func ToSyftModel(doc *spdx.Document2_2) (*sbom.SBOM, error) {
 
 	spdxIDMap := make(map[string]interface{})
 
+	src := source.Metadata{}
+	if doc.CreationInfo != nil {
+		src = extractSourceFromNamespace(doc.CreationInfo.DocumentNamespace)
+	}
+
 	s := &sbom.SBOM{
+		Source: src,
```

Comment:
Is this a required field in the struct? Should we only set this field `if doc.CreationInfo != nil`

Reply:
This is best effort to define `source.Scheme`, without `source.Scheme` we can't finish the conversion cycle.

Reply:
We discussed this one, I think this is good to keep; it could be iterated on in the future to try to determine the source type in other ways